connectors-home/modules/ROOT/partials/mq-disclaimer.adoc

```diff
@@ -29,4 +29,9 @@ To prevent message deletion until after processing, use the `AUTO` or `MANUAL` a
 // MQ Prefetch Issue note
 // tag::mqPrefetchLatency[]
 NOTE: Due to a known issue, using `prefetch` mode with `maxConcurrency="1"` set on the `flow` element might result in message-processing latency.
-// end::mqPrefetchLatency[] 
\ No newline at end of file
+// end::mqPrefetchLatency[]
+
+// MQ Prefetch maxLocalMessages
+// tag::mqPrefetchMaxLocalMsgs[]
+When using `prefetch` mode, set `maxLocalMessages` to a value greater than `1` to avoid message-retrieval errors.
```

Comment:
can we use something else instead of `message-retrieval errors`? what do you think about `message processing issues` ?

What if we combine the two tags - mqPrefetchLatency and mqPrefetchMaxLocalMsgs ?

Reply:
I can change it to "message-processing issues".

I'm not sure what you mean about combining the two tags. Do you want the content of both tags to appear everywhere the mqPrefetchLatency and mqPrefetchMaxLocalMsgs note appears?

For example, mqPrefetchLatency doesn't currently appear in the Connector reference.

Reply:
I think Both mqPrefetchLatency and mqPrefetchMaxLocalMsgs refer to `Prefetch` config and content should appear together (including connector reference)

Reply:
Everywhere? If so, I can merge this new one into the old one and add it to this PR.